Missing dependency when building sonic-workspace

### Describe your issue

The CMakeLists relies on [KWinDBusInterface](https://github.com/Sonic-DE/sonic-workspace/blob/36bb2cca3626e221a487446a913fb94574cf951b/CMakeLists.txt#L130) but that is not in sonic-win. Instead, sonic-win provides `KWinX11DBusInterface`.

### Steps to reproduce

1. Build/install sonic-win in a fresh environment
2. Try to build sonic-workspace
3. cmake fails

### What did you expect?

The build to work

### Additional context

```
CMake Error at CMakeLists.txt:130 (find_package):
  Could not find a package configuration file provided by "KWinDBusInterface"
  with any of the following names:

    KWinDBusInterfaceConfig.cmake
    kwindbusinterface-config.cmake

  Add the installation prefix of "KWinDBusInterface" to CMAKE_PREFIX_PATH or
  set "KWinDBusInterface_DIR" to a directory containing one of the above
  files.  If "KWinDBusInterface" provides a separate development package or
  SDK, be sure it has been installed.
```

### Extra fields

- [x] I have checked the existing issues
- [x] I'd like to work on this issue

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -127,7 +127,7 @@ set_package_properties(QALCULATE PROPERTIES
    TYPE REQUIRED)
 pkg_check_modules(DBus dbus-1 REQUIRED IMPORTED_TARGET)
 
-find_package(KWinDBusInterface CONFIG REQUIRED)
+find_package(KWinX11DBusInterface CONFIG REQUIRED)
 find_package(ScreenSaverDBusInterface CONFIG REQUIRED)
 
 # Used in kcm_autostart

